torch/testing/FileCheckBuilder.py

```diff
@@ -0,0 +1,54 @@
+import torch
+
+FileCheck = torch._C.FileCheck
+
+
+# Handle to serialize series of checks from python
+class FileCheckBuilder:
```

Comment:
Eventually I was envisioning this being used in text only - tests, which would require a parser. Although current usage doesn't require it. & agreed, builder should be move to C++

Reply:
I do not think we need a separate builder object since the methods can be directly written on the current object.  Our use case is different from FileCheck so I think we should wait until we actually want to use a file-only test before building a parser for it.